.env

```
@@ -0,0 +1,8 @@
+DB_HOST=10.250.0.103
+DB_NAME=easyevent
+DB_USER=easyevent
+DB_PASS=a[ez-4.wBhai48M8
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_PASS key should go before the DB_USER key

(UnorderedKey)